packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -36,13 +36,18 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 - Added PEM certificate support as an authentication method for logging into the API ML. [#2621](https://github.com/zowe/zowe-explorer-vscode/issues/2621)
 - Deprecated the `getUSSDocumentFilePath` function on the `IZoweTreeNode` interface as Zowe Explorer no longer uses the local file system for storing USS files. **No replacement is planned**; please access data from tree nodes using their [resource URIs](https://github.com/zowe/zowe-explorer-vscode/wiki/FileSystemProvider#operations-for-extenders) instead. [#2968](https://github.com/zowe/zowe-explorer-vscode/pull/2968)
 - **Next Breaking:** Changed `ProfilesCache.convertV1ProfToConfig` method to be a static method that requires `ProfileInfo` instance as a parameter.
+- Added the `onVaultUpdate` VSCode events to notify extenders when credentials are updated on the OS vault by other applications. [#2994](https://github.com/zowe/zowe-explorer-vscode/pull/2994)
```

Comment:
Wondering if we could tweak this sentence a bit. I'm getting stuck on why "VSCode events" is plural. If the plural is correct (and not a typo), can we rewrite this to say something like: 

- "Added the `onVaultUpdate` series/package/collection of VS Code events to ..." OR
- "Added VSCode events `onVaultUpdate` to notify ... "